### ISSUE TITLE
warmup api is not working?

### ISSUE THREAD

**Issue by mtikmani** [opendistro-for-elasticsearch/k-NN#237]
I have  KNN plugin built from source and added as a plugin to ES OSS 7.8.0. 

When I am running warmup api, I am hitting error. any pointers? 

```
GET /_opendistro/_knn/warmup/bugdata,budata_knn?pretty

{
  "error" : "no handler found for uri [/_opendistro/_knn/warmup/bugdata,budata_knn?pretty=&pretty=true] and method [GET]"
}

```

**Comment by mtikmani** [comment]
looks like warmup api is not yet present in official release build of knn(i.e 1.9.0). please correct if i am wrong. 

If my understanding is correct, is there a way I can manually load all the graphs post bulk indexing?

**Comment by vamshin** [comment]
@mtikmani, yes your understanding is correct. Warmup api available from 1.10.0.

> If my understanding is correct, is there a way I can manually load all the graphs post bulk indexing?

You will have to run few queries to load the graphs to memory. Probably run queries for a minute or two.
